component/authenticator/src/test/java/org/wso2/carbon/identity/application/authenticator/totp/services/TOTPAdminServiceTest.java

```diff
@@ -79,7 +79,7 @@ public void validateTOTPTest() throws Exception {
         userClaimValues.put(TOTPAuthenticatorConstants.SECRET_KEY_CLAIM_URL, encryptedSecretKey);
         doReturn(userClaimValues).when(mockUserStoreManager).getUserClaimValues(username + "@" + tenantDomain,
                 new String[]{TOTPAuthenticatorConstants.SECRET_KEY_CLAIM_URL}, null);
-        when(TOTPUtil.decrypt(anyString())).thenReturn(secretKey);
+        when(TOTPUtil.decryptSecret(anyString())).thenReturn(secretKey.getBytes());
```

Comment:
TOTPUtil.decrypt() method is now deprecated and we are no longer using it in our code base. And with the validateTOTP class we are now using the decreptSecret instead of decrypt method.